### PR TITLE
Feature/userstory 6 - edit function

### PR DESCRIPTION
**Pull request by AstridOttmann**

### CODE REVIEW

File: pages/words/[id].js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const router = useRouter();
+  const { id } = router.query;
+
+  const entry = translationList.find((translation) => {
+    return translation.id === id;
+  });
+  console.log("entry", entry);
+  console.log("id", id);
+
+  function handleCancelEdit() {
```
Comment (RobinWitt) [inline pages/words/[id].js]: Are you using this function only for the one button? Maybe you can just put `router.back()` directly in the onClick?

File: components/Form/index.js [hunk 2]
```diff
@@ -1,61 +1,80 @@
...
     const data = Object.fromEntries(formData);
 
-    onAddTranslations({
+    onSubmitEvent({
       word: data.word,
       language: data.language,
       translated: data.translated,
     });
 
+    if (isEditMode) {
```
Comment (RobinWitt) [inline components/Form/index.js:32]: Could you explain to me what this is doing?
Are you using the same form to add and change entries and the 'isEditMode' changes the behavior of the form?

File: public/store.js [hunk 3]
```diff
@@ -1,34 +1,39 @@
 import { atom, useAtom } from "jotai";
 
 const initialTranslations = [
-  { id: 1, word: "verjaardag", language: "Dutch", translated: "Geburtstag" },
-  { id: 2, word: "uitnodigen", language: "Dutch", translated: "einladen" },
+  { id: "1", word: "verjaardag", language: "Dutch", translated: "Geburtstag" },
```
Comment (RobinWitt) [inline public/store.js:4]: why did you need to change your IDs from numbers to strings?

**Review by RobinWitt — COMMENTED** [review]
The code looks good to me, I just left a couple of questions so I can understand what is going on.

File: components/Form/index.js (hunk 2 above)
Comment (AstridOttmann) [reply components/Form/index.js:32]: Exactly!

File: pages/words/[id].js (hunk 1 above)
Comment (AstridOttmann) [reply pages/words/[id].js]: Yes, you're right!

File: public/store.js (hunk 3 above)
Comment (AstridOttmann) [reply public/store.js:4]: Otherwise the form was not recognizing the entry .

**Review by RobinWitt — APPROVED** [review]

**Review by AGeck113 — APPROVED** [review]
